### ISSUE TITLE
Improve user messages for `cbuild setup` with compiler selection and layer is delivered in one step

### ISSUE THREAD

**Issue by ReinhardKeil** [Open-CMSIS-Pack/devtools#1708]
**Describe the bug**

To user message could be improved as explained below.

**To Reproduce**

Use this project [USBD.zip](https://github.com/user-attachments/files/16675747/USBD.zip)

```
>cbuild setup USB_Device.csolution.yml -S
error csolution: undefined variables in USB_Device.csolution.yml:
  - $Board-Layer$

error csolution: board 'STMicroelectronics::B-U585I-IOT02A' was not found
error csolution: processing context 'HID.Debug+B-U585I-IOT02A' failed
error csolution: compiler undefined, use '--toolchain' option or add 'compiler: <value>' to yml input, selectable values can be found in cbuild-idx.yml
error cbuild: exit status 2
```

A command line user is confused by unclear messages.

**Expected behavior**

Suggest this improvements: 

```
>cbuild setup USB_Device.csolution.yml -S
error csolution: undefined variables in USB_Device.csolution.yml:
  - $Board-Layer$
To resolve undefined variables, copy the settings from cbuild-idx.yml to csolution.yml

error csolution: compiler undefined, use '--toolchain' option or add 'compiler: <value>' to csolution.yml, possible values in cbuild-idx.yml 
```

> **Note:** small modifications to the error messages.

**Environment:**

- CMSIS-Toolbox 2.5.0+p16-g5f15bb17
- OS: Windows

**Comment by soumeh01** [comment]
Unable to reproduce it using `cmsis-toolbox 2.6.0`. It shows the same results as expected

```
$ cbuild setup USB_Device.csolution.yml -S -l all
error csolution: undefined variables in USB_Device.csolution.yml:
  - $Board-Layer$
To resolve undefined variables, copy the settings from cbuild-idx.yml to csolution.yml
error cbuild: exit status 2
```

**Comment by jkrech** [comment]
It just had not been validated with the released 2.6.0 version. Closing
